utils/src/main/java/com/cloud/utils/ssh/SshHelper.java

```diff
@@ -206,4 +216,87 @@ public static void scpTo(String host, int port, String user, File pemKeyFile, St
                 conn.close();
         }
     }
+
+    /**
+     * It gets a {@link Session} from the given {@link Connection}; then, it waits
+     * {@value #WAITING_OPEN_SSH_SESSION} milliseconds before returning the session, given a time to
+     * ensure that the connection is open before proceeding the execution.
+     *
+     * @param conn
```

Comment:
@DaanHoogland are you talking about the `@param`, `@return` and `@throws`? It was auto-generated by Eclipse, I can remove them. I agree that they do not add value (even because in this case it has any explanation), just didn't pay attention to them.